tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -2934,6 +2934,16 @@ port::Status CudnnSupport::DoConvolve(
           "This configuration has potential integer overflow in "
           "cuDNNv5 and cuDNNv6. See b/68264959.");
     }
+    if (CUDNN_VERSION < 8000) {
```

Comment:
int8-to-float convolution with CUDNN_CONVOLUTION_FWD_ALGO_IMPLICIT_GEMM clamps out at [-128,127], but shouldn't.  We need to check against it.  CUDNN 8.0 is an estimation.  Once the fix is scheduled, we may need to adjust this condition.